python/tskit/trees.py

```diff
@@ -5840,6 +5891,28 @@ def kc_distance(self, other, lambda_=0.0):
         """
         return self._ll_tree_sequence.get_kc_distance(other._ll_tree_sequence, lambda_)
 
+    def count_topologies(self, sample_sets=None):
+        """
+        Generates the same distribution of topologies as :meth:`Tree.count_topologies`
+        but for every tree in a tree sequence. For use on a tree sequence this method is
```

Comment:
How exactly is this combined across all the trees? Is it added together by tree? Weighted by span?

Reply:
They are not combined, it produces a generator that yields the per-tree result. Can clarify